scripts/translation/lib/GitLogParser.php

```diff
@@ -42,22 +42,37 @@ static function parseInto( string $lang , RevcheckFileList & $list )
             // commit message
             if ( str_starts_with( $line , '    ' ) )
             {
-                // commits marked with `[skip-revcheck]` are skiped in full
+                // commits with this mark are ignored
                 if ( stristr( $line, '[skip-revcheck]' ) !== false )
                     $skip = true;
                 continue;
```

Comment:
Maybe we should make this more lenient and accept ``[skip revcheck]``, ``[revcheck skip]``, and ``[revcheck-skip]`` ?

Reply:
This is easy to implement. But will further diverge this new code from the two+ other revchecking code, as it may cause the revcheck.html produced from this code diverge from other revcheck variations, making it a little harder to future migrations.

I suggest first replacing/removing duplicated code before changing it further.

Algo, the only documented way to skip is `[skip-revcheck]`.

Reply:
Ack, I was wondering why the revcheck code was being duplicated :)

Reply:
In a distant future, this new folder can be submodules imported from doc-base and web-doc, and revcheck front-ends may use this unified revcheck back-end as the single source of revcheck data.

Reply:
As this PR merged, I will first start a new PR to address specifically this item, and then after this, start the move to remove revcheck code duplication from doc-base.

Reply:
Thank you!